README.md

```diff
@@ -124,12 +124,29 @@ step.
 
 #### Importing
 
-This step is responsible for creating a new database and filling it. If any
+This step is responsible for preparing the database and filling it. If any
 exporting strategies have been used those must have run first, or their outputs
 must have been downloaded if they are being shared/hosted somewhere.
 
 Factory-based strategies generate data during this process.
 
+##### Reset modes
+
+By default any existing database will be removed, ensuring that a fresh database
+is created for the imported data. This is expected to be the most common case
+for local development, but may not always be suitable.
```

Comment:
Can we add an example of the command line argument being used here, just to demonstrate that this is controlled in that way? Doesn't have to be right here, just in an appropriate place around here.

Reply:
Thoughts on beef078?